rendering_resource_manager_service/deployment/rrm/populateRRM.txt

```diff
@@ -2,20 +2,7 @@
 # of the rendering resource manager.
 
 # Livre
-curl -curl --dump-header - -H "Accept:application/json" -H "Content-Type:application/json" -X POST --data '{"id": "livre", "command_line": "livre", "modules": "", "environment_variables": "", "process_rest_parameters_format": "--zeroeq-http-server ${rest_hostname}:${rest_port}", "scheduler_rest_parameters_format": "--zeroeq-http-server $SLURMD_NODENAME:${rest_port}", "graceful_exit": "True", "wait_until_running": "True"}' http://localhost:8383/rendering-resource-manager/v1/config/
-
-# RTNeuron
```

Comment:
This is only a informational file, and since RTNeuron is not open-source, it does not make any sense to expose it for now. That's why I removed it.